tests/platform_tests/test_platform_info.py

```diff
@@ -13,6 +13,7 @@
 from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
 from tests.common.utilities import wait_until
 from thermal_control_test_helper import *
+from tests.platform_tests.cli.test_show_platform import check_vendor_specific_psustatus
```

Comment:
This works. But it is not a good practice to import a function from another test script. Can we make check_vendor_specific_psustatus common?

Reply:
Good suggestion. I think it's better to relocate this method to cli/util.py. That's a more common place. 
@jleveque Is that OK?

Reply:
I'm surprised I didn't encounter this myself.

Eventually, I want to do away with this method of checking vendor-specific status and use JSON (or other data language) files to represent each platform, so that it is a universal solution.

For now, I suggest moving the function back into test_platform_info.py and replacing https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/cli/test_show_platform.py#L121 to a comment: `# TODO: Compare against expected platform-specific output`

Reply:
Updated. Please help to review. Thanks!

Reply:
Looks goo. Thanks!